pwa/src/components/footer/Footer.css

```diff
@@ -0,0 +1,18 @@
+.utrecht-page-footer {
+    display: flex;
+}
+
+.utrecht-page-footer > .utrecht-unordered-list {
+    flex: 1;
+}
+
+.utrecht-page-footer .utrecht-heading-4 {
+    color: white;
+}
+
+.utrecht-page-footer .utrecht-unordered-list__item {
```

Comment:
Since we're not showing the bullets, we might want to remove the padding, like so:

`padding-inline-start: 0;`

This way, the redundant indentation is removed.